instructor/ui/sketch_tool.js

```diff
@@ -60,7 +61,7 @@ function load_sketch_tool(x_upper_bound, tick_values) {
 
 
     function mouseDownEvent (event) {
-        path = new Path();
+        path = new paper.Path();
```

Comment:
What's the significance of this change? Is the new `Path` tied to that paper instance?